WordPress/Classes/Services/PeopleService.swift

```diff
@@ -0,0 +1,98 @@
+import Foundation
+
+class PeopleService {
+    let remote: PeopleRemote
+    let siteID: Int
+
+    private let context = ContextManager.sharedInstance().mainContext
+
+    init(blog: Blog) {
+        remote = PeopleRemote(api: blog.restApi())
+        siteID = blog.dotComID() as Int
+    }
+
+    func refreshTeam(completion: (Bool) -> Void) {
+        remote.getTeamFor(siteID,
+            success: {
+                (people) -> () in
+
+                self.mergeTeam(people)
```

Comment:
I took a different approach and made `PeopleService` a `struct`. I'm not 100% sure, but from what I understand, it is now a _value type_, so there are no reference cycles: the callback just gets a copy of the service.

I also think this matches what I wanted more closely. I think I changed it to a `class` because I planned to keep track of requests in progress, but I don't think that's happening in the short term